install-config.yaml

```diff
@@ -23,7 +23,7 @@ networking:
     hostPrefix: 23
   machineNetwork:
   - cidr: 192.168.126.0/24
-  networkType: OpenShiftSDN
+  networkType: OVNKubernetes
```

Comment:
>  you might need to increase memory resource to 10G

If there are concerns about resource usage, can this be mentioned/tracked in this PR?

Reply:
This anyway need to propagate to crc codebase, but I can mention it here also.

Reply:
Imo it's better to make people aware of the impact of this PR *before* we take it in, rather than learning about it after the fact through a crc PR.